exporter/opentelemetry-exporter-prometheus-remote-write/tests/test_prometheus_remote_write_exporter.py

```diff
@@ -0,0 +1,416 @@
+
+        expected_min_timeseries = self.exporter._create_timeseries(
+            value_observer_record, "testname_min", 1.0
+        )
+        expected_max_timeseries = self.exporter._create_timeseries(
+            value_observer_record, "testname_max", 5.0
+        )
+        expected_sum_timeseries = self.exporter._create_timeseries(
+            value_observer_record, "testname_sum", 8.0
+        )
+        expected_count_timeseries = self.exporter._create_timeseries(
+            value_observer_record, "testname_count", 3.0
+        )
+        expected_last_timeseries = self.exporter._create_timeseries(
+            value_observer_record, "testname_last", 2.0
+        )
+        timeseries = self.exporter._convert_from_value_observer(
+            value_observer_record
+        )
+        self.assertEqual(timeseries[0], expected_min_timeseries)
+        self.assertEqual(timeseries[1], expected_max_timeseries)
+        self.assertEqual(timeseries[2], expected_sum_timeseries)
+        self.assertEqual(timeseries[3], expected_count_timeseries)
+        self.assertEqual(timeseries[4], expected_last_timeseries)
+
+    # Ensures quantile aggregator is correctly converted to timeseries
+    # TODO: Add test_convert_from_quantile once method is implemented
+
+    # Ensures timeseries produced contains appropriate sample and labels
+    def test_create_timeseries(self):
+        def create_label(name, value):
+            label = Label()
+            label.name = name
+            label.value = value
+            return label
+
+        sum_aggregator = SumAggregator()
+        sum_aggregator.update(5)
+        sum_aggregator.take_checkpoint()
+        export_record = ExportRecord(
+            Counter("testname", "testdesc", "testunit", int, None),
+            get_dict_as_key({"record_name": "record_value"}),
+            sum_aggregator,
+            Resource({"resource_name": "resource_value"}),
+        )
+
+        expected_timeseries = TimeSeries()
+        expected_timeseries.labels.append(create_label("__name__", "testname"))
+        expected_timeseries.labels.append(
+            create_label("resource_name", "resource_value")
+        )
+        expected_timeseries.labels.append(
+            create_label("record_name", "record_value")
+        )
+
+        sample = expected_timeseries.samples.add()
+        sample.timestamp = int(sum_aggregator.last_update_timestamp / 1000000)
+        sample.value = 5.0
+
+        timeseries = self.exporter._create_timeseries(
+            export_record, "testname", 5.0
+        )
+        self.assertEqual(timeseries, expected_timeseries)
+
+
+class TestExport(unittest.TestCase):
+    # Initializes test data that is reused across tests
+    def setUp(self):
+        self.exporter = PrometheusRemoteWriteMetricsExporter(
+            endpoint="/prom/test_endpoint"
+        )
+
+    # Ensures export is successful with valid export_records and config
+    @patch("requests.post", return_value=Mock())
```

Comment:
I think that this should work, please give it a try:

```suggestion
    @patch("requests.post")
```

The patched object should be replaced by a [`MagicMock`](https://docs.python.org/3/library/unittest.mock.html#unittest.mock.patch) in this case.

Reply:
It worked. Removed unnecessary specified return_value.